test/lib/accounts.spec.js

```diff
@@ -0,0 +1,206 @@
+import * as accounts from '../../src/lib/accounts'
+
+const mockKonnector = {
```

Comment:
You're right, but I would prefer a naming format like `mockedObject` so, since it's not about an action here. What do you think?